docs/glossary.rst

```diff
@@ -0,0 +1,278 @@
+==================
+Terms and concepts
+==================
+
+Concepts and terminology may overlap with existing technologies,
+whereas the requirements specified here are intended to be agnostic to specific solutions.
+This document assumes the following definitions of potentially overloaded key words.
+
+.. glossary::
+
+    task
```

Comment:
Requesting clarification from work load management team
* distinctions between task representations and task implementations
* scope (single-instruction-multiple-data? decomposability / hierarchy of representations of schedulable executable objects?)
* precondition, invariant, post-condition?